### PR TITLE
Make the VH1.com background dark

### PR DESCRIPTION
**Pull request by torma616**

### CODE REVIEW

**Comment by Gusted** [comment]
Hi could you provide a example site?

**Comment by NguyenASang** [comment]
@Gusted Maybe this site https://www.vh1.com ?

**Comment by torma616** [comment]
@Gusted, @NguyenASang has it right. https://www.vh1.com. Darkreader makes the default dark text light but doesn't do anything to the background, rendering the text unreadable. This PR will make the background dark.

**Comment by Gusted** [comment]
Really weird, I just get redirected to their facebook page 🤷🏽

**Comment by torma616** [comment]
It might be accessible in US only

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -125,6 +125,14 @@ div[class^='c5'] .gm-style
 
 ================================
 
+*.vh1.com
+
+CSS
+.module-container {
+    background-color: var(--darkreader-neutral-background) !important;
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:133]:
```suggestion
}

```

**Review by Gusted — APPROVED** [review]

**Pull request merged**